src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './styles.scss';
+
+export default function Sidebar() {
+  return (
+    <section className='sidebar-container'>
```

Comment:
Could you give some sample screenshots for more illustration?

Reply:
Notice the sidebar

![Screen Shot 2564-01-21 at 10 46 42](https://user-images.githubusercontent.com/32289827/105277471-0e8a3680-5bd6-11eb-9f8a-950d7842da37.png)

Now in small screen, it stay besides footer

![Screen Shot 2564-01-21 at 10 47 06](https://user-images.githubusercontent.com/32289827/105277518-26fa5100-5bd6-11eb-88e4-f0ec473a979d.png)

Reply:
Fixing this would be a waste of time since we don't know what designs are going to be. Can we keep it still for now and adjust them according to the designs later?

Reply:
Okay 👌